mumbleBot.py

```diff
@@ -320,30 +321,35 @@ def launch_next(self):
 
     def download_music(self, url):
         url_hash = hashlib.md5(url.encode()).hexdigest()
-        path = var.config.get('bot', 'tmp_folder') + url_hash + ".mp3"
-        ydl_opts = {
-            'format': 'bestaudio/best',
-            'outtmpl': path,
-            'noplaylist': True,
-            'writethumbnail': True,
-            'updatetime': False,
-            'postprocessors': [{
-                'key': 'FFmpegExtractAudio',
-                'preferredcodec': 'mp3',
-                'preferredquality': '192',
-            }]
-        }
-        video_title = ""
-        with youtube_dl.YoutubeDL(ydl_opts) as ydl:
-            for i in range(2):
-                try:
-                    info_dict = ydl.extract_info(url)
-                    video_title = info_dict['title']
-                except youtube_dl.utils.DownloadError:
-                    pass
-                else:
-                    break
-        return path, video_title
+        path = var.config.get('bot', 'tmp_folder') + url_hash + ".%(ext)s"
+        mp3 = path.replace(".%(ext)s", ".mp3")
```

Comment:
Why did you split this line in two ?

Reply:
I'm not sure what you mean? You need to give .%(ext)s to youtube-dl but need to check for .mp3 and return it as the path back from the function.

Reply:
And to add to this, doing this instead of str.replace(".mp3", ".%(ext)s") seemed nicer

Reply:
I didn't know that youtube-dl need %(ext)s value.